projects/igniteui-angular/src/lib/card/card.component.ts

```diff
@@ -245,7 +260,6 @@ export class IgxCardComponent {
      * <igx-card type="outlined"></igx-card>
      * ```
      */
-    @HostBinding('class.igx-card')
     @Input()
     public type: IgxCardType | string = IgxCardType.ELEVATED;
```

Comment:
Shouldn't this be `IgxCardType.OUTLINED` by default since we're changing the behavior?